CMakeLists.txt

```diff
@@ -533,6 +533,11 @@ target_compile_options(giada PRIVATE ${COMPILER_OPTIONS})
 
 include(GNUInstallDirs)
```

Comment:
Should we move this and the `install` command below in the `if(DEFINED OS_LINUX)` block?

Reply:
no idea. I don't know CMake.

Reply:
The abstractions offered by GNUInstallDirs may also make sense on macOS, but if you build e.g. app bundles for that OS, it is better moved below a Linux only conditional statement.